internal/fnruntime/exec.go

```diff
@@ -59,6 +61,10 @@ func (f *ExecFn) Run(r io.Reader, w io.Writer) error {
 	cmd.Stdout = w
 	cmd.Stderr = &errSink
 
+	for k, v := range f.Env {
+		cmd.Env = append(cmd.Env, fmt.Sprintf("%v=%v", k, v))
```

Comment:
Are exec functions unable to access env vars today? Why is this change necessary for the wasm stuff?

Reply:
The exec functions can access env vars today. But we want to provide additional env var when running with node.js runtime.
This change makes `ExecFn` not only to inherit env var from the host (current behavior) but also allow us to provide additional env var. It makes `ExecFn` more flexible to use as a library. This change will not affect end users directly.